input/definitions/MeasureReport/StructureDefinition-countQuantity.json

```diff
@@ -0,0 +1,78 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "countQuantity",
+  "url": "http://hl7.org/fhir/StructureDefinition/countQuantity",
```

Comment:
Following extension naming conventions, because this is an extension that is only usable on MeasureReport, the extension id and URL should reflect that:
id: measurereport-countQuantity
url: http://hl7.org/fhir/StructureDefinition/measurereport-countQuantity